tests/scalers/cassandra.test.ts

```diff
@@ -43,6 +47,7 @@ test.before(t => {
     t.is('1', cassandraClientReadyReplicaCount, 'Cassandra client is not in a ready state')
 
     // create table
+    console.log("create table")
```

Comment:
As I mentioned on the issue, somewhere around here should be some checks, because even if the cassandra and cassandra client pods are ready, the Cassandra itself is not fully started. So would be nice to run some CQLquery here in wait loop, to check once cassandra is up. The query could just check status or something.